spec/policies/request_for_comment_policy_spec.rb

```diff
@@ -122,6 +177,18 @@
               it_behaves_like 'grants access to admins and authors only'
             end
           end
+
+          permissions(:report?) do
+            before do
+              codeocean_config = instance_double(CodeOcean::Config)
+              allow(CodeOcean::Config).to receive(:new).with(:code_ocean).and_return(codeocean_config)
+              allow(codeocean_config).to receive(:read).and_return({
+                content_moderation: {report_emails: ['report@example.com']},
+              })
+            end
```

Comment:
This part is often copied and unchanged. Can't we extract it to an outer scope and just set the `report_emails` array to be empty or to contain email addresses using `let`? Then, we could save some repetitions, I would assume.